tracer/build/_build/Build.Steps.cs

```diff
@@ -2213,7 +2215,7 @@ private void CopyDumpsTo(AbsolutePath root)
     {
         if (Directory.Exists(TempDirectory))
         {
-            foreach (var dump in GlobFiles(TempDirectory, "coredump*"))
+            foreach (var dump in GlobFiles(TempDirectory, "coredump*", "*.dmp"))
```

Comment:
Interesting that these ones end up in the `TempDirectory`, whereas below they're in the `TracerDirectory` 🤔 🤷‍♂️